balloc/balloc.c

```diff
@@ -547,9 +560,9 @@ static int sb_update(struct m0_balloc *bal, struct m0_sm_group *grp)
 	return M0_RC(rc);
 }
 
-static int balloc_sb_write(struct m0_balloc            *bal,
-			   struct m0_balloc_format_req *req,
-			   struct m0_sm_group          *grp)
+static int balloc_sb_write(struct m0_balloc		  *bal,
```

Comment:
Usually, we only use "TAB" at the beginning of a line to do indentation.
But in line itself, we use spaces to align the indentifiers.

Reply:
Done